.github/workflows/conformance-k8s-kind-network-policies.yaml

```diff
@@ -0,0 +1,258 @@
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+        with:
+          ref: ${{ steps.vars.outputs.sha }}
+          persist-credentials: false
+
+      - name: Wait for images to be available
+        timeout-minutes: 30
+        shell: bash
+        run: |
+          for image in cilium-ci operator-generic-ci hubble-relay-ci ; do
+            until docker manifest inspect quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/$image:${{ steps.vars.outputs.sha }} &> /dev/null; do sleep 45s; done
+          done
+
+      - name: Install Cilium
+        run: |
+          cilium install --wait ${{ steps.vars.outputs.cilium_install_defaults }}
+
+      - name: Run Kubernetes sig-network conformance test
+        run: |
+          # output_dir
+          mkdir -p _artifacts
+
+          # get kubeconfig to pass to the e2e binary
+          kind get kubeconfig --name ${{ env.cluster_name }} > _artifacts/kubeconfig.conf
+
+          # Kubernetes e2e tests use ginkgo and tags to select the tests that should run based on two regex, focus and skip:
+          # Focus tests:
+          # \[Conformance\]|\[sig-network\]: Conformance tests are defined by the project to guarantee a consistent behaviour and some mandatory features on all clusters
+          #                                  sig-network tests are defined by sig-networkto guarantee a consistent behaviour on all the the k8s network implementations
+          # Skipped tests:
+          # Disruptive|Serial : require to run in serial and perform disruptive operations on clusters (reboots, ...)
+          # Federation|PerformanceDNS : unrelated sig-network tests
+          # Feature : skip features that are not GA, however, some of them should be enabled, per example [Feature:ProxyTerminatingEndpoints]
+          # DualStack : only with dualstack clusters
+          # KubeProxy|kube-proxy : kube-proxy specifics
+          # LoadBalancer|GCE|ExternalIP : require a cloud provider, some of them are GCE specifics
+          # Aggregator : Flaky, https://github.com/cilium/cilium/issues/24622.
+          # same.port.number.but.different.protocols|HostPort|should.serve.endpoints.on.same.port.and.different.protocols : #9207
+          # rejected : Kubernetes expect Services without endpoints associated to REJECT the connection to notify the client, Cilium silently drops the packet
+          # externalTrafficPolicy : needs investigation
+
+          # Run tests
+          export KUBERNETES_CONFORMANCE_TEST='y'
+          export E2E_REPORT_DIR=${PWD}/_artifacts
+          /usr/local/bin/ginkgo --nodes=25                \
+            --focus="\[Conformance\]|\[sig-network\]"     \
+            --skip="Feature|Federation|PerformanceDNS|DualStack|Disruptive|Serial|KubeProxy|kube-proxy|ExternalIP|LoadBalancer|GCE|Aggregator|rejected|externalTrafficPolicy|HostPort|same.port.number.but.different.protocols|should.serve.endpoints.on.same.port.and.different.protocols"   \
```

Comment:
As said in the PR message this is the only line that is different from the `conformance-k8s-kind.yaml` test.
So are adding Network policy tests.

Then my question is why not modify the `conformance-k8s-kind.yaml` file and remove `Netpol|NetworkPolicy`  from that file and instead yet another workflow file?

Reply:
the main problem with network policies is that use to flake a lot because of the implementations, if we merge both jobs we don't have a good way of differentiating

Reply:
in kubernetes per example we only run network policy tests on presubmits based on the code modified, but we run them periodically every X hours, that is the best strategy IMHO

Reply:
Thank you, for the clarification.